docs/stylesheets/extra.css

```diff
@@ -153,3 +153,14 @@ html .md-footer-meta .md-footer-meta__inner a {
 .md-typeset .admonition.note .admonition-title::before {
     color: #c8aeae;
 }
+
+/* show the copy-to-clipboard button only if .copy-to-clipboard is specified */
+.codehilite:not(.copy-to-clipboard) .md-clipboard {
+    display: none;
+}
+
+/* do not copy the terminal prompt `$` */
+pre > code > .gp {
+    -webkit-user-select: none;
```

Comment:
Note that on Safari the terminal prompts `$` are still being copied from multiline snippets, with the exception of the one in the first line (see this related [WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=80159))